src/components/QuranReader/TajweedBar/TajweedBar.tsx

```diff
@@ -0,0 +1,79 @@
+import { useEffect, useRef, useState } from 'react';
+
+import classNames from 'classnames';
+import useTranslation from 'next-translate/useTranslation';
+
+import styles from './TajweedBar.module.scss';
+
+import ChevronDownIcon from '@/icons/chevron-down.svg';
+
+const TAJWEED_COLORS = [
+  'edgham',
+  'mad-2',
+  'mad-2-4-6',
+  'mad-4-5',
+  'mad-6',
+  'ekhfa',
+  'qalqala',
+  'tafkhim',
+];
+
+const TajweedColors = () => {
+  const [showTajweedBar, setShowTajweedBar] = useState(true);
+  const [height, setHeight] = useState(0);
+  const ref = useRef(null);
+
+  const { t } = useTranslation('quran-reader');
+
+  const toggle = () => {
+    setShowTajweedBar(!showTajweedBar);
+  };
+
+  useEffect(() => {
+    setHeight(ref.current.clientHeight);
+  }, [ref.current.clientHeight]);
+
```

Comment:
This is throwing an exception
<img width="975" alt="Screenshot 2024-06-21 at 10 43 47 AM" src="https://github.com/quran/quran.com-frontend-next/assets/15169499/a0ac8f57-6511-419c-b030-66d11ba92978">